### PR TITLE
Switch to using Vec<u8> for ble_ident

### PR DESCRIPTION
**Pull request by rschulman**

### CODE REVIEW

File: src/mdl_reader.rs [hunk 1]
```diff
@@ -97,14 +97,10 @@ pub fn establish_session(
                 value: "the device did not transmit a central client uuid".to_string(),
             })?;
 
-    // TODO this must be wrong -- it's encoded as a hex string, but then advertised as raw bytes
-    let mut ble_ident = hex::encode(ble_ident);
```
Comment (sbihel) [inline src/mdl_reader.rs:101]: `initialise_session` needs the same change

**Comment by sbihel** [comment]
(this needs manual testing which I'm hoping I can do by tomorrow)

**Comment by Ryanmtate** [comment]
@simon Any chance we can merge this before #22 ?

**Comment by sbihel** [comment]
Closing as this now doesn't have the full fixes for ble_ident. Will do the work directly in #22

**Pull request closed without merging**